stripe/api/stripe.api

```diff
@@ -6544,6 +6546,14 @@ public abstract interface class com/stripe/android/view/StripeEditText$ErrorMess
 	public abstract fun displayErrorMessage (Ljava/lang/String;)V
 }
 
+public final class com/stripe/android/view/StripeEditText$StripeEditTextState$Creator : android/os/Parcelable$Creator {
```

Comment:
Revert?

Reply:
hmm I rebased master and reran api dump, this is still generated. I think it's generated by`@Parcelize`, where in java a `CREATOR` class needs to be manually created

Reply:
Ah interesting, good to know.